website/content/guides/1.7-template-upgrade.mdx

```diff
@@ -85,7 +108,7 @@ packer {
     required_plugins {
         comment = {
             version = ">=v0.2.23"
-            source = "sylviamoss/comment"
+            source = "github.com/sylviamoss/comment"
```

Comment:
Make source contain hostname; not sure we're gonna force it but if we do it's easier not to have to come back and change it.

Reply:
btw I wanted to use your username here but I need a valid release in your repo. If you want to do it, we can change it here later. I think it would nice since the plugin is yours not mine, I don't want to take any credits even if there's a fork flag in my repo. 👀

Reply:
No worries at all! It works as-is. Let's leave it be.